Assets/HotmaxEditorVR/telekinesisControl.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class telekinesisControl : MonoBehaviour
+{
+
+    private float initialPadYPosition;
+    private float currentPadYPos;
+
+    private float distToController;
+    [SerializeField]
+    private float scrollSpeed = .2f;
+    private float scrollDistance = 0;
+
+
+    private void OnEnable()
+    {
+        //this gets the initial offset of the object to the controller
+        //the purpose being to start the object off at the same place it was in before selecting it
+        initialPadYPosition = inputManager.selectorHand.GetAxis().y;
+        distToController = Vector3.Distance(transform.position, inputManager.hand2.transform.position);
+        scrollDistance = 0;
+    }
+
+    // Update is called once per frame
+    void Update ()
+    {
+        init.rotationGizmos.SetActive(false);
```

Comment:
this could be set just once onEnable and be the same, no?

Reply:
oh, yeah. got it